app_remote/flaskr/static/js/piiAppClient.js

```diff
@@ -0,0 +1,20 @@
+class PiiAppClient {
+  constructor (host) {
+    this.host = host
+  }
+
+  replaceTicketId (payload) {
```

Comment:
This function looks to be returning a promise. It's not strictly necessary, however I think it might be better to prefix the "async" keyword to this function

- Not strictly necessary as it returns a promise anyway
- Easier in that users can immediately see that it returns a promise
- This will make it return a Promise that returns a Promise..... which I believe js copes with by ignoring intermediate Promises... I could be wrong here. If i'm wrong on this point please ignore comment

Reply:
Agreed 👍  it's better to have this prefix as you mentioned would benefit users as they get immediate notification of what it supposes to return. Regarding to the 3rd point, found an example [here](https://javascript.info/async-await#async-functions) not a very good one, but according to it they are the same.

Reply:
Great! I was remembering correctly then! haha js isn't my preferred language, so it's a long time between bouts :)